src/lib/y2storage/planned/has_size.rb

```diff
@@ -213,6 +203,26 @@ def adjusted_size_after_ceil(device, space_size, align_grain)
         def total_weight(devices)
           devices.map(&:weight).reduce(0, :+)
         end
+
+        # Checks if there are eough space at all
+        # @raise RuntimeError if there is not enough space
+        def check_size(devices, space_size)
+          needed_size = DiskSize.sum(devices.map(&:min))
+          return unless space_size < needed_size
+
+          log.error "not enough space: needed #{needed_size}, available #{space_size}"
+          raise RuntimeError
+        end
+
+        # counts minimal size for each device
```

Comment:
I wouldn't say this "counts" the minimal sizes. The name of the method is not that good. I would say it's more like:

```ruby 
# Sets sizes of the given devices to their minimal valid value
# @return [Array<Planned::Device>] modified devices with size adjusted to rounded minimum size
def init_sizes(device, rounding)
  [...]
end
```